projects/plugins/social/src/js/components/header/index.js

```diff
@@ -1,23 +1,61 @@
-import { Container, Col, H3 } from '@automattic/jetpack-components';
+import { Container, Col, H3, Text, Spinner, SocialIcon } from '@automattic/jetpack-components';
+import { useSelect, useDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
-import illustration from './illustration.svg';
+import { Icon, edit, lifesaver } from '@wordpress/icons';
+import { STORE_ID } from '../../store';
 import styles from './styles.module.scss';
 
-const Header = () => (
-	<Container horizontalSpacing={ 3 } horizontalGap={ 7 } className={ styles.container }>
-		<Col sm={ 4 } md={ 4 } lg={ 5 }>
-			<H3 mt={ 2 }>{ __( 'Share & Grow', 'jetpack-social' ) }</H3>
-			<p className={ styles.title }>
-				{ __(
-					'Jetpack Social allows you to share your WordPress posts with your social networks.',
-					'jetpack-social'
-				) }
-			</p>
-		</Col>
-		<Col sm={ 4 } md={ 4 } lg={ 6 } className={ styles.illustration }>
-			<img src={ illustration } alt="" />
-		</Col>
-	</Container>
+const Actions = ( { actions } ) => (
+	<div className={ styles.actions }>
+		{ actions.map( ( { link, label, icon } ) => (
+			<Text key={ label }>
+				<a href={ link } className={ styles.action }>
+					{ icon && <Icon icon={ icon } size={ 16 } /> }
+					{ label }
+				</a>
+			</Text>
+		) ) }
+	</div>
 );
+const Header = () => {
+	useDispatch( STORE_ID ).getSharesCount();
```

Comment:
If this is to live here then I think it needs to be wrapped in a `useEffect`
```suggestion
	useEffect( () => { useDispatch( STORE_ID ).getSharesCount() }, [] );
```

Without it the component will be rendered again as the shares count is received and it will get stuck in a loop requesting the shares count constantly.

I think it's good to have this in the store and a way to request it, but I wonder if for now we should add this into the initial state? Maybe something for a follow-up PR